ansible/roles/fanout/templates/sonic_deploy_cisco_8101_202205.j2

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
Why we need a new template for 8101? Can you integrate it to common sonic fanout config template?

Reply:
We use a specific template because the generic template doesn't work, and we don't know why it doesn't work. Also this template can accept port config that differ from hwsku